lib/widgets/mxc_image.dart

```diff
@@ -233,9 +233,16 @@ class _MxcImageState extends State<MxcImage>
 
   Widget placeholder(BuildContext context) =>
       widget.placeholder?.call(context) ??
-      const Center(
-        key: Key(placeholderKey),
-        child: CircularProgressIndicator.adaptive(),
+      SizedBox(
+        width: widget.width ?? 256,
```

Comment:
remove the default size, it will make image resize

Reply:
deleted